index.html

```diff
@@ -1,12 +1,56 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Beginning Web Development</title>
     <link rel="stylesheet" href="main.css">
 </head>
+
 <body>
 
-<script src="main.js"></script>
+    <!-- NAV BAR (this is to test using github/pull requests)-->
+
+    <!-- HERO -->
+
+    <!-- ABOUT -->
+
+    <!-- BLOG -->
+
+    <!-- FOOTER -->
+
+    <footer>
+        <div class="row">
+            <div class="col col--w30">
+                <a href="#" target="_blank"><img src="./images/bwd-logo-darkmode.svg" alt="BWD logo"></a>
```

Comment:
I'm not sure if 'target="_blank"' should be used here since clicking the logo leads to the same/home page.